src/ui/server/helpers/sanitise-data/index.test.ts

```diff
@@ -42,7 +43,17 @@ describe('server/helpers/sanitise-data', () => {
 
   describe('STRING_NUMBER_FIELDS', () => {
     it('should return an explicit array of field IDs that are string fields that could have a pure number value', () => {
-      const expected = [CREDIT_PERIOD_WITH_BUYER, DESCRIPTION, COMPANY_NUMBER, PHONE_NUMBER, GOODS_OR_SERVICES];
+      const expected = [
+        CREDIT_PERIOD_WITH_BUYER,
+        DESCRIPTION,
+        COMPANY_NUMBER,
+        PHONE_NUMBER,
+        GOODS_OR_SERVICES,
+        ADDRESS_LINE_1,
+        ADDRESS_LINE_2,
+        POSTCODE,
+        BROKER_NAME,
```

Comment:
Do we want broker name to potentially be a number?  We don't do this for first/last name in account creation (maybe we should?)

Reply:
I am just thinking what if the broker name was a number for some weird reason

Reply:
Valid scenario - remember speaking to design about this, we should allow anything for a name. Will create a ticket to update account first/last name fields 👍